pkg/roachpb/genbatch/main.go

```diff
@@ -9,13 +9,12 @@
 // licenses/APL.txt.
 
 // This file generates batch_generated.go. It can be run via:
-//    go run -tags gen-batch gen_batch.go
-
-// +build gen-batch
+//    go run -tags genbatch genbatch/main.go
```

Comment:
Did you test this, are you sure it works?

Reply:
Yeap, I tried with `make generate` and also manually in my local machine (running command `go run -tags genbatch genbatch/main.go`) ant it is working